doc/INSTALL.md

```diff
@@ -22,74 +22,115 @@ For the following Linux distributions and package managers 0.8 packages are avai
 For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/show/home:bekun:devel/elektra)
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 
-### Ubuntu-Bionic
+We also provide more recent versions and bleeding-edge versions in our repositories:
```

Comment:
"more recent" and "bleeding-edge" unclear. Isn't it latest release and latest build of master?